### PR TITLE
Update backend watch to run a list, reconcile and then watch

### PR DESCRIPTION
**Pull request by jamestalton**

### CODE REVIEW

File: backend/config/LOG_LEVEL [hunk 1]
```diff
@@ -1 +1 @@
-debug
\ No newline at end of file
+trace
```
Comment (KevinFCormier) [inline backend/config/LOG_LEVEL:1]: Was it intentional to commit this change?

Comment (jamestalton) [reply backend/config/LOG_LEVEL:1]: yes

**Pull request merged**